script/Ciel/Day 10/10_01_CIEL10_7.txt

```diff
@@ -926,47 +996,51 @@ And yet, my heart feels at ease.
 [sha:e2b80ef3ffec55d553c9aacc433a90157d4db97d]{
 -- Page 579, Offset 28702.
 -- 昨日のように沈みこむ事も、自己嫌悪にまみれる事もない。
-I don't feel depressed, nor am I wallowing in self-hatred like yesterday.
+I don't feel depressed like I did yesterday,
+ nor am I wallowing in self-hatred.
 }
 [sha:c16ee51ab2ca7b5797dcf616d54310507d6935d1]{
 -- Page 580, Offset 28703.
 -- 共にいられない事は初めから予感していた。
-I had the presentiment from the start that we wouldn't be able to be together―――
+From the very beginning, I'd understood on some level that we could never be together.
 }
 [sha:ad21bfdc69cd75bebc96e7cc90eb844480efd42d]{
 -- Page 580, Offset 28704.
 -- 俺たちは同じ世界、同じ時間に生きてはいないと。
-And that we aren't living in the same world and in the same period.
+We don't live in the same world, or even the same era.
 }
 [sha:4fe900f964ab18ce1f1279df314b9acb0f3a1850]{
 -- Page 580, Offset 28705.
 -- そもそも人間は誰であれいずれ別れる。それがあんなに苦しかったのは、きちんとした別れができなかったからだ。
-By nature, all humans will eventually have to go their own way. The reason it felt so painful is that we didn't properly part ways.
+Everyone, no matter who they are, has to say goodbye at some point.
+ The reason it had felt so painful before is because we hadn't properly said our farewells.
 }
 [sha:eba1910caf780021c12e612dbe6a79ad4abc2db0]{
 -- Page 581, Offset 28706.
 -- 今は違う。俺は先輩と笑顔で別れる事ができた。
-This time is different. I was able to say goodbye to Senpai with a smile on my face.
+This time was different. I was able to say goodbye to Senpai with a smile on my face.
```

Comment:
Think it could connect better. Present tense because it's his feelings in the moment, not the farewell itself.
>But now it's different.